include/libvfio-user.h

```diff
@@ -569,6 +580,17 @@ typedef struct {
 
 } vfu_migration_callbacks_t;
 
+/*
+ * Completes a pending migration state transition. Calling this function when
+ * there is no pending migration state transition results in undefined
+ * behavior.
+ *
+ * @vfu_ctx: the libvfio-user context
+ * @err: 0 for success, -1 on error having set errno.
```

Comment:
this argument should surely just be the errno?

Reply:
ok